fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/block/FabricBlockSettings.java

```diff
@@ -315,7 +315,10 @@ public FabricBlockSettings breakByHand(boolean breakByHand) {
 	/**
 	 * Please make the block require a tool if you plan to disable drops and slow the breaking down using the
 	 * incorrect tool by using {@link FabricBlockSettings#requiresTool()}.
+	 *
+	 * @deprecated Replaced by {@code mineable} tags. See fabric-mining-level-api-v1 for further details.S
 	 */
+	@Deprecated(forRemoval = true)
```

Comment:
Technically shouldn't be `forRemoval` I think, unless it's broken beyond salvation?

Reply:
This PR actually fixes it (it's broken in 1.17) so I'll take out the forRemoval.

Reply:
if this fix is hacky, you can still consider just leaving it noop and remove it later with the annotation